--- src/java/com/ctc/wstx/sr/InputElementStack.java~	2013-04-16 23:24:26.000000000 +0200
+++ src/java/com/ctc/wstx/sr/InputElementStack.java	2013-06-20 14:03:54.950052611 +0200
@@ -843,9 +843,13 @@
      * element.
      */
     public int addDefaultAttribute(String localName, String uri, String prefix,
-                                   String value) throws XMLStreamException
+                                   String value)
     {
-        return mAttrCollector.addDefaultAttribute(localName, uri, prefix, value);
+        try {
+            return mAttrCollector.addDefaultAttribute(localName, uri, prefix, value);
+        } catch (XMLStreamException e) {
+            throw new RuntimeException(e);
+        }
     }
 
     /*
